app/src/pages/inside/productVersionPage/linkedTestCasesTab/tagList/messages.js

```diff
@@ -33,8 +33,4 @@ export const messages = defineMessages({
     id: 'LinkedTestCasesTab.noTagsAdded',
     defaultMessage: 'No tags added',
   },
-  removeTag: {
```

Comment:
I don't think we should remove it. In component in UI kit the string is hardocded in English. By providing it here we make sure it is translated to other languages